src/Domains/Guild/Leads/DataTransferObject/LeadRotation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Guild\Leads\DataTransferObject;
+
+use Spatie\LaravelData\Data;
+use Kanvas\Companies\Models\Companies;
+use Kanvas\Apps\Models\Apps;
+
+class LeadRotation extends Data
+{
+    public function __construct(
+        public Apps $app,
+        public Companies $company,
+        public string $name,
+        public ?string $leads_rotations_email = null,
```

Comment:
@FredPeal you can use calmecase and use the attribute  to allow use to insert the value with _

Reply:
these 2
#[MapInputName(SnakeCaseMapper::class)]
#[MapOutputName(SnakeCaseMapper::class)]